quotes.yaml

```diff
@@ -23,7 +23,14 @@
 
 - :c: A computer lets you make more mistakes faster than any other invention in human history, with the possible exceptions of handguns and tequila.
   :a: Mitch Ratcliffe
+  
+- :c: But we are HACKERS and hackers have black terminals with green font colors
+  :a: J. Nunemaker   
 
+- :c: Hope That Firewall works because your screwed...
```

Comment:
I'd vote this to be left out, because it doesn't seem anything special. @moll agree?